packages/gatsby-telemetry/src/error-helpers.js

```diff
@@ -1,11 +1,18 @@
+const { sep } = require(`path`)
+
 // Removes all user paths
-const cleanPaths = str => {
-  // TODO: Recursively remove path sections
-  const localPathRegex = new RegExp(
-    process.cwd().replace(/[-[/{}()*+?.\\^$|]/g, `\\$&`),
-    `g`
-  )
-  return str.replace(localPathRegex, `$PWD`)
+const cleanPaths = (str, separator = sep) => {
+  const stack = process.cwd().split(separator)
```

Comment:
should `process.cwd()` better as parameter of the helper to have a pure function here?